lib/middleware/assignActiveLicence.js

```diff
@@ -0,0 +1,44 @@
+const { acquisitionCtxClient } = require('@financial-times/kat-client-proxies');
+
+const RE_FTCOM = /^FT\.com\s*/i;
```

Comment:
What does `RE_FTCOM` mean?

Reply:
Not sure, but it's just a regex to strip out the FT.com from the product value returned by the acquisition context request. Rather annoyingly we get back value like 'FT.com Standard'. 'FT.com Premium' rather than a product code or 'Premium'. 'Standard' etc.